drivers/gpio/Kconfig.sx1509b

```diff
@@ -0,0 +1,44 @@
+# Kconfig.pcal9535a - PCAL9535A GPIO configuration options
+#
+#
+# Copyright (c) 2018 Aapo Vienamo
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+menuconfig GPIO_SX1509B
+	bool "SX1509B I2C GPIO chip"
+	depends on GPIO && I2C
+	default n
+	help
+	  Enable driver for SX1509B I2C GPIO chip.
+
+if GPIO_SX1509B
+
+config GPIO_SX1509B_INIT_PRIORITY
+	int
+	default 70
+	prompt "Init priority"
+	help
+	  Device driver initialization priority.
+
+config GPIO_SX1509B_DEV_NAME
+	string "SX1509B GPIO chip Device Name"
+	default "GPIO_P0"
+	help
+	  Specify the device name for the SX1509B I2C GPIO chip.
+
+config GPIO_SX1509B_I2C_ADDR
+	hex "SX1509B GPIO chip I2C slave address"
+	default 0x3e
+	help
+	  Specify the I2C slave address for the SX1509B I2C GPIO chip.
+
+config GPIO_SX1509B_I2C_MASTER_DEV_NAME
```

Comment:
The device name and address configs should come from dts if possible. Please add bindings in dts/bindings/sensor